sam/methods.go

```diff
@@ -23,6 +23,11 @@ func (s *SamAln) GetChromEnd() int {
 	return s.GetChromStart() + runlength
 }
 
+func (s *SamAln) UpdateLift(c string, start int, end int) {
+	s.RName = c
+	s.Pos = int64(start + 1)
```

Comment:
What is the SamToBed convention? It's not start plus 1 is it? I imagine it's something like the end coordinate of the reference the read is aligned to, but that gets complicated because I think now you have to take into account the cigar run length information.

Reply:
Never mind I found it. Dan implemented the logic in GetChromEnd() in methods.go